packages/flutter_webrtc/tizen/src/flutter_video_renderer.cc

```diff
@@ -0,0 +1,178 @@
+#include "flutter_video_renderer.h"
+
+namespace flutter_webrtc_plugin {
+
+FlutterVideoRenderer::FlutterVideoRenderer(TextureRegistrar* registrar,
+                                           BinaryMessenger* messenger)
+    : registrar_(registrar) {
+  texture_ =
+      std::make_unique<flutter::TextureVariant>(flutter::PixelBufferTexture(
+          [this](size_t width,
+                 size_t height) -> const FlutterDesktopPixelBuffer* {
+            return this->CopyPixelBuffer(width, height);
+          }));
+
+  texture_id_ = registrar_->RegisterTexture(texture_.get());
+
+  std::string event_channel =
+      "FlutterWebRTC/Texture" + std::to_string(texture_id_);
+  event_channel_.reset(new EventChannel<EncodableValue>(
+      messenger, event_channel, &StandardMethodCodec::GetInstance()));
+
+  auto handler = std::make_unique<StreamHandlerFunctions<EncodableValue>>(
```

Comment:
This part (and more parts in this and other files) has [recently been refactored](https://github.com/flutter-webrtc/flutter-webrtc/commit/8f183ed17a3f0d75eb05c397bb88cc60312d7127) in the upstream (22 days ago). When did you last fetch their latest commit?

Reply:
Current version is 0.9.13, I will update to 0.9.17

Reply:
https://github.com/flutter-webrtc/flutter-webrtc/pull/1198